Cargo.lock

```diff
@@ -580,3 +580,74 @@ dependencies = [
  "winapi-build 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[metadata]
```

Comment:
I'm not sure here. Since `Cargo.lock` is tracked in the git repo, I just added it.